docs/getting-started/cli-getting-started.md

```diff
@@ -119,12 +134,14 @@ For more information, see [Writing scripts](../user-guide/cli-using-writing-scri
 
 ## Next steps
 
-You successfully installed Zowe CLI, issued your first commands, and wrote a simple script! Next, you might want to perform the following tasks:
+You successfully installed Zowe CLI, issued your first commands, and wrote a simple script! 
+
+Next, you might want to perform the following tasks:
 
 - Issue the `zowe --help` command to explore the product functionality, or review the online [web help](../user-guide/cli-using-displaying-help.md).
-- Learn how to configure Zowe CLI [run Zowe CLI in daemon mode](../user-guide/cli-using-using-daemon-mode.md). **Daemon mode** significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process.
+- Learn how to configure Zowe CLI [to run in daemon mode](../user-guide/cli-using-using-daemon-mode.md). **Daemon mode** significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process.
 - Learn about [configuring environment variables](../user-guide/cli-configuringcli-ev.md) to store configuration options.
-- Learn about [integrating with API Mediation Layer](../user-guide/cli-using-integrating-apiml.md).
+- Learn about [using API Mediation Layer](../user-guide/cli-using-integrating-apiml.md).
 - Learn about how to [write scripts](../user-guide/cli-using-writing-scripts.md) and integrate them with automation server, such as Jenkins.
 - See what [plug-ins are available](../user-guide/cli-extending.md) for the CLI.
```

Comment:
I don't think this link lists the available plug-ins anymore
https://docs.zowe.org/stable/user-guide/cli-extending

Maybe we should point people to:
https://docs.zowe.org/stable/user-guide/cli-installplugins

However, the `cli-installplugins` is missing the MQ and FTP plug-ins 😅 
![image](https://github.com/user-attachments/assets/c32443c5-6dee-4724-b118-dc9e5e1d7a0d)

Reply:
Made some edits to that page.